package.json

```diff
@@ -46,6 +46,7 @@
     "appium-base-driver": "3.x",
     "appium-espresso-driver": "^1.0.0",
     "appium-fake-driver": "^0.x",
+    "appium-flutter-driver": "^0",
```

Comment:
Sorry I'm still confused, as this package itself is `appium-flutter-driver`, isn't this one a self-depedenccy (it depends on itself?

Reply:
Do you mean you would like to work `appium-flutter-driver` as Appium when it is installed with `-g` flag?

Or should a user specify `package.json` with `appium` and `appium-flutter-driver` if he/she would like to use flutter-driver as local dependencies?
I mean in Appium desktop app case, appium-desktop's package.json should have both appium and appium-flutter-driver as its packlage.json instead of only `appium`.

(This change is for your appium fork, btw)

Reply:
ah yeah i thought this repo was `appium-flutter-driver` :D